tests/tck/features/optimizer/PrunePropertiesRule.feature

```diff
@@ -19,12 +19,12 @@ Feature: Prune Properties rule
       | 33            |
       | 41            |
     And the execution plan should be:
-      | id | name           | dependencies | operator info                                                                                    |
-      | 8  | Project        | 4            |                                                                                                  |
-      | 4  | AppendVertices | 3            | {  "props": "[{\"props\":[\"age\"],\"tagId\":9}]" }                                              |
-      | 3  | Traverse       | 7            | {  "vertexProps": "", "edgeProps": "[{\"props\":[\"_dst\", \"_rank\", \"_type\"],\"type\":3}]" } |
-      | 7  | IndexScan      | 2            |                                                                                                  |
-      | 2  | Start          |              |                                                                                                  |
+      | id | name           | dependencies | operator info                                                                                     |
+      | 8  | Project        | 4            |                                                                                                   |
+      | 4  | AppendVertices | 3            | {  "props": "[{\"props\":[\"age\"],\"tagId\": 3}]" }                                              |
```

Comment:
maybe we should remove the `tagId` from the tests since it depends on the runtime environment.

Reply:
Agree, better display name instead of Id. Could do it in later PR.